api/location/models.py

```diff
@@ -3,15 +3,28 @@
 
 from helpers.database import Base
 
+
 from utilities.utility import Utility
+import enum
+
+
+class Country_type(enum.Enum):
```

Comment:
In python, classes are never names with underscores, always use CountryType syntax -underscores are used in naming functions